cmd/minishift/cmd/hostfolder/mount.go

```diff
@@ -32,8 +31,8 @@ var (
 	mountAll bool
 )
 
-var hostfolderMountCmd = &cobra.Command{
-	Use:   "mount HOSTFOLDER_NAME",
+var mountCmd = &cobra.Command{
+	Use:   "mount HOST_FOLDER_NAME",
 	Short: "Mounts the specified host folder to the running OpenShift cluster.",
```

Comment:
Here it is cluster...

Reply:
This is the one I actually forgot to change.